_opensearch/search-backpressure.md

```diff
@@ -0,0 +1,221 @@
+search_backpressure.<br>&nbsp;&nbsp;&nbsp;&nbsp;search_shard_task.<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;heap_moving_average_window_size | 100 | The number of previously completed search shard tasks to consider when calculating the rolling average of heap usage.
+search_backpressure.<br>&nbsp;&nbsp;&nbsp;&nbsp;search_shard_task.<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;cpu_time_millis_threshold | 15,000 ms | The CPU usage threshold (in milliseconds) for a single search shard task before it is considered for cancellation.
+search_backpressure.<br>&nbsp;&nbsp;&nbsp;&nbsp;search_shard_task.<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;elapsed_time_millis_threshold | 30,000 ms | The elapsed time threshold (in milliseconds) for a single search shard task before it is considered for cancellation.
+
+## Search Backpressure Stats API
+Introduced 2.4
+{: .label .label-purple }
+
+You can use the [nodes stats API operation]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/text-analyzers/#how-to-use-text-analyzers) to monitor server-side request cancellations.
+
+#### Sample request
+
+To retrieve the stats, use the following request:
+
+```json
+GET _nodes/stats/search_backpressure
+```
+
+#### Sample response
+
+The response contains server-side request cancellation statistics:
+
+```json
+{
+  "_nodes": {
+    "total": 1,
+    "successful": 1,
+    "failed": 0
+  },
+  "cluster_name": "runTask",
+  "nodes": {
+    "T7aqO6zaQX-lt8XBWBYLsA": {
+      "timestamp": 1667409521070,
+      "name": "runTask-0",
+      "transport_address": "127.0.0.1:9300",
+      "host": "127.0.0.1",
+      "ip": "127.0.0.1:9300",
+      "roles": [
+         
+      ],
+      "attributes": {
+        "testattr": "test",
+        "shard_indexing_pressure_enabled": "true"
+      },
+      "search_backpressure": {
+        "search_shard_task": {
+          "resource_tracker_stats": {
+            "heap_usage_tracker": {
+              "cancellation_count": 34,
+              "current_max_bytes": 1203272,
+              "current_avg_bytes": 700267,
+              "rolling_avg_bytes": 1156270
+            },
+            "cpu_usage_tracker": {
+              "cancellation_count": 318,
+              "current_max_millis": 731,
+              "current_avg_millis": 303
+            },
+            "elapsed_time_tracker": {
+              "cancellation_count": 310,
+              "current_max_millis": 1305,
+              "current_avg_millis": 649
+            }
+          },
+          "cancellation_stats": {
+            "cancellation_count": 318,
+            "cancellation_limit_reached_count": 97
+          }
+        },
+        "mode": "enforced"
+      }
+    }
+  }
+}
+```
+
+### Response fields
+
+The response contains the following fields.
+
+Field Name | Data Type | Description
+:--- | :--- | :---
+search_backpressure | Object | Contains statistics about search backpressure.
```

Comment:
For consistency, either delete the verb "contains" in line 173 and 174 or add a verb to each description in lines 175-177.